bin/webpack-configs.js

```diff
@@ -643,13 +623,22 @@ const getStylingConfig = ( options = {} ) => {
 				cacheGroups: {
 					editorStyle: {
 						// Capture all `editor` stylesheets and editor-components stylesheets.
-						test: ( module = {} ) =>
-							module.constructor.name === 'CssModule' &&
-							( findModuleMatch( module, /editor\.scss$/ ) ||
-								findModuleMatch(
-									module,
-									/[\\/]assets[\\/]js[\\/]editor-components[\\/]/
-								) ),
+						test: ( module = {}, { moduleGraph } ) => {
+							if ( ! module.type.includes( 'css' ) ) return false;
```

Comment:
Nit: I think it's better to always use clauses. This will keep consistency with our codebase and IMO makes code easier to read.
```suggestion
							if ( ! module.type.includes( 'css' ) ) {
								return false;
							}
```